egs2/cmu_arctic/tts1/run_pre_fine.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+fs=16000
+n_fft=1024
+n_shift=256
+win_length=null
+
+inference_config=conf/decode.yaml
+
+# g2p=g2p_en # Include word separator
+g2p=g2p_en_no_space # Include no word separator
+
+
+pretrain_stage=true
+adapt_stage=true
+# If you want to run only one of the stages (e.g., the adaptation stage),
+# set "false" to the one you don't want to run (e.g., the pre-training stage)
+
+if [[ ${pretrain_stage} == "true" ]]; then
+    spk=all
+    train_set=${spk}_train_no_dev
+    valid_set=${spk}_dev
+    test_sets=${spk}_eval
+    train_config=conf/tuning/train_transformer_pre.yaml
```

Comment:
Does it make sense to train transformer-TTS with multi-speaker data using no speaker information?